Package.swift

```diff
@@ -0,0 +1,38 @@
+// swift-tools-version:5.2
+import PackageDescription
+
+let package = Package(
+    name: "DevCycle",
+    platforms: [
+        .iOS(.v12),
+        .macOS(.v10_12)
```

Comment:
ohh do we actually support macOS?

Reply:
this was to quell an error with trying to build